app/forms/income/TradingAllowanceAmountFormProvider.scala

```diff
@@ -14,20 +14,20 @@
  * limitations under the License.
  */
 
-package forms
-
-import javax.inject.Inject
+package forms.income
 
 import forms.mappings.Mappings
 import play.api.data.Form
 
+import javax.inject.Inject
+
 class TradingAllowanceAmountFormProvider @Inject() extends Mappings {
```

Comment:
Is this form correct? Why is it accepting bigDecimal, should it not be an enumerable as to me the page in the prototype is this https://income-tax-submission.herokuapp.com/v06/6-2-5/self-employment/income/allowance-amount-selection ?